languages/ruby/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,49 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of "nil" in Ruby.
+
+## Learning objectives
+
+- Know that `true` is a Ruby object used to represent logical `true`
+- Know that everything except `false` and `nil` is truthy
+- Know that `false` is a Ruby object used to represent logical `false`
+- Know that `false` is falsey
+- Know that `nil` is a Ruby object used to represent "nothingness"
```

Comment:
Probably move this right below the initial mention of false as well…
- True is logical true
- false is logical false
- nil is an object that represents nothing.

(to paraphrase the listing above)